tests/entropy.c

```diff
@@ -175,6 +190,7 @@ int main(int argc, char **argv) {
                 }
 
                 // decode
+                gettimeofday(&tv2, NULL);
```

Comment:
I think you need one of these straight after the compression part as well.  At the moment you're including the time to do some printing, and in the SIMD tests a memcmp(), which would make the compression times look worse than they really are.